sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ChangeFeedFetcher.java

```diff
@@ -62,7 +62,7 @@ public ChangeFeedFetcher(
         } else {
             DocumentClientRetryPolicy retryPolicyInstance = client.getResetSessionTokenRetryPolicy().getRequestPolicy();
             String collectionLink = PathsHelper.generatePath(
-                ResourceType.DocumentCollection, changeFeedState.getContainerRid(), false);
+                ResourceType.DocumentCollection, changeFeedState.getContainerRid(), false, null);
```

Comment:
if you add an overload `PathHelper.generatePath` with operationType = null default. then you don't have to modify many other files.

Reply:
Done